test/models/changeset/pull_request_test.rb

```diff
@@ -41,6 +41,40 @@
     end
   end
 
+  describe ".valid_webhook" do
+    let(:webhook_data) do
+      {
+        number: 1,
+        pull_request: {
+          state: 'open',
+          body: 'pr description [samson review]'
+        },
+        github: {
+          action: 'opened'
+        }
+      }.with_indifferent_access
+    end
+
+    it "is not valid if the github action is labeled" do
```

Comment:
would be nice to have more user-level description like `is invalid for a PR that had its label changed`